packages/components/package.json

```diff
@@ -41,13 +38,29 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.1.0",
     "@testing-library/user-event": "^13.2.1",
+    "@types/dom-mediacapture-record": "^1.0.11",
     "@types/html-truncate": "^1.2.1",
     "@types/mime": "^2.0.3",
     "@types/react": "^17.0.34",
     "@types/react-dom": "^17.0.11",
-    "@types/react-intl": "^2.3.18",
     "@types/react-slick": "^0.23.5",
     "jest-css-modules-transform": "^4.3.0",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
React is a dev dependency?

Reply:
Having `react` as a dev dependency and a peer dependency allows for any project using this library to not necessarily rely on v17.0.2 but rather use "any compatible" version they are already using.

For example, you might want to include this component library into your react app that runs v17.2.0. In this case, it would make no sense to also install 17.0.2 since the lib is compatible with any version above 17 (as per de peer dependency).